app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if parts[2].endswith("/"):
```

Comment:
Consider adding a check to ensure that the `parts` list has at least three elements before accessing `parts[2]`. This will prevent potential `IndexError` if the command string does not contain enough parts.